### ISSUE TITLE
Ipad not Load more than once

### ISSUE THREAD

**Issue by CaioMatias** [pisi/Reel#55]
Hi, i have a page with 6 car colors, and for each color i want to load a different reel.
In Desktop Safari is working fine, but on iPad just the first works.

Any idea?

Here is the page:
http://taxilabs.com.br/projetos/ogilvy/jac-j6/

Thanks!

**Comment by pisi** [comment]
Hi Matias,

I checked the page and so far have no idea why it won't load.

But I noticed a thing which may have something to do with it. I first checked on desktop Chrome and as you appear to have some sort of Flash movie below the Reel, I also clicked on the blocked movie (I block Flash in my browser) and it loaded fine. So I went to the iPad and at the first glance I noticed, that the iPad version featured Reel's default thin black band preloader, whereas the desktop version had its own spinning red one. But when clicked a color, the red spinner showed up and that was it.

Now the interesting part - I also happened to refresh the desktop browser, but this time without allowing the Flash movie. And what I did end up with was the very same endlessly spinning red spinner.

Since iPad has no Flash, that indeed looks quite suspicious. So I would advice to explore the portion of the code, which takes care about Flash (not) being on the target machine.

Anyway, really nice Reel use I must say! :)

.pisi
